index.html

```diff
@@ -0,0 +1,638 @@
+                    <li class="orangeText white artCategory">MOST POPULAR</li>
+                    <li class="orangeText latestTitle">Lightkey Pro uses AI to know what you're going to type before you type it</li>
+                  </ul>
+                </div>
+                <img src="Images/latest1.jpg" alt="">
+              </div>
+              <div class="list">
+                <ul class="maText orangeBullets">
+                  <li class="catListArt">CD Projekt Rekt: Cyberpunk 2077 tanks studio's share price nearly 50%</li>
+                  <li class="catListArt">CD Projekt stock down 20% since Cyberpunk reviews hit the internet</li>
+                  <li class="catListArt">Airbnb to be bigger than eBay after its long-awaited IPO</li>
+                  <li class="catListArt">Tesla just set a new record high. Now it wants to dump $5B in stock</li>
+                  <li class="catListArt">Meanwhile, sports and gaming stocks are pumping to record highs</li>
+                </ul>
+              </div>
+            </div>
+            <div class="category">
+              <h2 class="orangeTitle">Hard Fork</h2>
+              <div class="cat1 catCont">
+                <div class="catFreeArea">
+                </div>
+                <div class="articleHeader">
+                  <ul class="noDecoration">
+                    <li class="white artCategory">MOST POPULAR</li>
+                    <li class="white artTitle">Twitter shares for 7% in pre-market trading after Trump's removal</li>
+                    <li class="white credits">Ivan Mehta · 7 days ago</li>
+                  </ul>
+                </div>
+              </div>
+              <div class="latest lightpink">
+                <div class="latestText">
+                  <ul class="noDecoration">
+                    <li class="orangeText white artCategory">MOST POPULAR</li>
+                    <li class="orangeText latestTitle">Lightkey Pro uses AI to know what you're going to type before you type it</li>
+                  </ul>
+                </div>
+                <img src="Images/latest1.jpg" alt="">
+              </div>
+              <div class="list">
+                <ul class="maText orangeBullets">
+                  <li class="catListArt">CD Projekt Rekt: Cyberpunk 2077 tanks studio's share price nearly 50%</li>
+                  <li class="catListArt">CD Projekt stock down 20% since Cyberpunk reviews hit the internet</li>
+                  <li class="catListArt">Airbnb to be bigger than eBay after its long-awaited IPO</li>
+                  <li class="catListArt">Tesla just set a new record high. Now it wants to dump $5B in stock</li>
+                  <li class="catListArt">Meanwhile, sports and gaming stocks are pumping to record highs</li>
+                </ul>
+              </div>
+            </div>
+          </div>
+        </section>
+        <section class="latestsDeals">
+          <h2 class="orangeTitle">Latest Deals</h2>
```

Comment:
## Check HTML validator
- It's throwing some errors in these lines 535, 596, 634
![image](https://user-images.githubusercontent.com/24902525/105259891-b2ce9780-5b5a-11eb-9d7a-d6250fdb3814.png)